### PR TITLE
Upload artifacts to Releases after build

### PR DESCRIPTION
**Pull request by WillowSauceR**
Build pre-built packages in Action and release them to Release on version update

### CODE REVIEW

**Comment by Torrekie** [comment]
looks good, wait for @LNSSPsd to review

**Comment by WillowSauceR** [comment]
The effect of the latest changes can be previewed [HERE](https://github.com/WillowSauceR/PhoenixBuilder/releases/) . In addition, download links for files in the Github Release can be added to install.sh for the user to select a download source, like this:
Changes at install.sh:
ChangesAt_install sh
Output of install.sh:
OutputOf_install sh
Download speed from Official website:
OfficialSourceDwonloadSpeed
Download speed from Github Releases:
GithubSourceDwonloadSpeed

**Comment by Torrekie** [comment]
as of 3.4.1 release, we statically linked zlib and readline into FastBuilder, Android users no longer have to download external packages for program to run

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -196,6 +228,30 @@ jobs:
...
+          fb-upload.tar.gz
+          build/*
+          build/gui/*
+      
+    - name: Upload build directory to Action Artifact
+      uses: actions/upload-artifact@main
+      with:
+        name: fb-upload
+        path: build
+
```
Comment (LNSSPsd) [inline .github/workflows/build.yml:254]: Uploading stuff to Github Action's storage costs the 2GB free usage,
so I would prefer to only publish them in the Releases page.

**Review by LNSSPsd — APPROVED** [review]
Approve. But the step uploading stuff to the Github Actions will be removed.

**Pull request merged**